get-poetry.py

```diff
@@ -30,7 +30,7 @@
 import tempfile
 
 from contextlib import closing
-from contextlib import contextmanager
+from contextlib import contextmanager, suppress
```

Comment:
this install script is deprecated, but still technically supports python2. let's revert this change, but the rest look good

Reply:
To clarify further, we're considering it frozen and not subject to modification outside critical defects like a security issue. Even code formatting is not being applied.